velox/connectors/hive/HiveConfig.cpp

```diff
@@ -149,7 +149,10 @@ bool HiveConfig::isFileColumnNamesReadAsLowerCase(const Config* session) const {
 }
 
 bool HiveConfig::isPartitionPathAsLowerCase(const Config* session) const {
-  return config_->get<bool>(kPartitionPathAsLowerCaseSession, true);
```

Comment:
The original code is already acquiring the value from session, I think your change has the same logic as the original code

Reply:
@kewang1024 Ke, there is no notion of "session" in Velox. What does that mean here?

Reply:
@mbasmanova @kewang1024 You can refer to this [discussion](https://github.com/facebookincubator/velox/issues/7659) and [PR](https://github.com/facebookincubator/velox/pull/7725) , which provides a detailed description of the concept of `session` in `HiveConfig`.

Reply:
So for a connector, its config comes from two sources
- connector config file (Per connector)
- QueryCtx (Per query)

the QueryCtx is the same as session property in presto so I think we named it as session here